Applications/Embedding/README.md

```diff
@@ -0,0 +1,39 @@
+# Embedding
+
+This application contains embedding layer example with two example model
```

Comment:
Can we update this description to also include multi-input scenario?

Reply:
Updated